specs/commandline/policy.md

```diff
@@ -2,9 +2,9 @@
 
 ## Description
 
-As part of signature verification workflow of signed OCI artifacts, users need to configure trust policy configuration file to specify trusted identities that signed the artifacts, the level of signature verification to use and other settings. For more details, see [OCI trust policy specification and examples](https://github.com/notaryproject/specifications/blob/main/specs/trust-store-trust-policy.md#oci-trust-policy).
+As part of signature verification workflow of signed OCI artifacts, users need to configure trust policy configuration file to specify trusted identities that signed the artifacts, the level of signature verification to use and other settings. For more details, see [trust policy specification and examples](https://github.com/notaryproject/specifications/blob/v1.1.0/specs/trust-store-trust-policy.md#trust-policy).
```

Comment:
The link is 404 page not fund

Reply:
Yes, I'm actually aware of this, but since we will release the `v1.1.0` spec recently, I kept it.

Reply:
Updated to `v1.1.0-rc.1` to make the links work. We will need to update them to `v1.1.0` once the stable version of the spec is released.